kamrons-game.py

```diff
@@ -2,13 +2,31 @@
 from time import sleep
 import sys
 
+
+#sleep times:
```

Comment:
It's customary for constants to be in ALL CAPS so when we're reading the code, we can tell they're constants that are hard coded, and not for example function parameters or other variables that can be changed by the user or the programming.

Otherwise, well done.

Reply:
k

Reply:
done